docs/features/software-catalog/system-model.md

```diff
@@ -107,10 +107,6 @@ product or use-case, share the same entity types in their APIs, and integrate
 well with each other. Other domains could be “Content Ingestion”, “Ads” or
 “Search”.
 
-## Current status
-
-Backstage currently supports Components and APIs.
-
```

Comment:
Might be a bit early as long as the documentation is missing 😆

Reply:
I would for sure welcome an update to the UML a bit higher up which shows only the three "core" elements, as I'd now suspect it's not three but five with system + domain? Also helps to pitch the info to my user base as well.

There still may be a delineation of core vs extended, but maybe even if this UML gets moved up a section to reflect the current full state of the model.  As they say: a picture is worth a thousand words!

![image](https://user-images.githubusercontent.com/33203301/103904022-b712a380-50ca-11eb-9c48-643b40351fbf.png)

Reply:
Not sure, it was part of the old version of the image and was explicitly removed there.

Reply:
Sorry, what did you mean?  (apols, just didn't know which "there" you're referring)

e.g., it's in your fork still (and actively published up on backstage.io/docs from `master` based on my previous screen shot above)

![image](https://user-images.githubusercontent.com/33203301/103923239-dd443d80-50e2-11eb-80d9-cd1187a7f54e.png)

Reply:
I guess all I'm suggesting is, adding the context to the model is awesome, and as a user I find the picture to be really helpful - sort of a standard architectural model 🗺️ that might show the new kinds that are being introduced, that's all! 😁

Reply:
Yeah a graph like this should absolutely be front and center in the model pages!

Reply:
Ok, I will search for the original one, we should have it somewhere here in GIT

Reply:
This is the original system model: 
![image](https://user-images.githubusercontent.com/648527/103990497-9b260500-5191-11eb-9887-1478d29fc4f0.png)

We could use it, but I wonder if we want to use different terms for the arrows. I think it would be best to use the wording from relations, what do you think @freben ?

Reply:
On the other hand the section explicitly states the three "core entities", I wonder if the new entities are also "core entities"? If yet, than we should rewrite the section completely

Reply:
The reason I split it originally is that the domain and system modelling might differ quite a bit between organizations, and they might be optional. As such, it could be an advantage to have a set of "core entities" that we are strict about. Most functionality and plugins for Backstage could then operate on those core entities, without relying on any specific way of laying out systems/domains/applications or other abstractions. That way, they are easier to share between organizations. Not sure how important that concern is at this point.

Reply:
I think that it is valid. We could also keep the image as-is, and show how system and domains interact with it in a second image in the "Ecosystem modeling" section below.

Reply:
I went for two diagrams and reworked them to be draw.io SVG files to edit them easier later. What I changed:
* Changed the colors, previously they were kind of random. Now the core entities are green and the others orange.
* I changed all names of the relations. For resources I went with `attachesTo` even though we haven't talked about that yet. But the files are now easy to edit, we can change that later.

Reply:
I just realized ... I wonder what our friendly neighborhood svg optimization bot will do with this auxiliary draw.io data -_-

Reply:
😱

Reply:
Alternative: I export the files without drawio data and put drawio files beside them? But we can check that once the image bot did his work.

Reply:
Yeah, just keep some backups around locally :)